src/util/tr.js

```diff
@@ -499,6 +496,29 @@ const _tr = (msg, {
       const body = "<div class='wzrkPPdscr' style='color:" + textColor + "'>" + descriptionText + '<div></td></tr></table></div>'
       html = css + title + body
     }
+    if (displayObj.preview && displayObj['custom-editor']) {
+      iframe.sandbox = 'allow-scripts allow-popups allow-popups-to-escape-sandbox'
+      /* eslint-disable no-cond-assign */
+      var properties = {}
+      const styleRegex = /<style[^>]*>([^<]*)<\/style>/g
+      const ctClassRegex = /\.CT_(?:Box|Banner)\s*{([^}]*)}/g // Regex to match either .CT_Box or .CT_Banner
```

Comment:
what about `interstitial` ?

Reply:
Its working fine!

Reply:
there are still console errors, can we add handling for interstitial as well?

Reply:
These errors are related to click tracking, which we are unable to perform in a sandboxed iframe.